src/deepLink/deepLink.ts

```diff
@@ -0,0 +1,8 @@
+export enum DeepLinkTypes {
+	Branch = 'branch',
+	Commit = 'commit',
+	Remote = 'remote',
+	Tag = 'tag',
+}
+
+export type DeepLinkType = DeepLinkTypes;
```

Comment:
Wanted to use the union of the enum values as a type, and using the enum as a type directly was a bit weird because it uses plural phrasing i.e. `DeepLinkTypes`, but I suppose I could just rename the enum and use it directly when I'm referring to it as a type. Would that work better?

Reply:
Updated in commit: [Move constants/enums out](https://github.com/gitkraken/vscode-gitlens/pull/2471/commits/fc0fffd26e9264597ec8f08f41bc2aaf6404e9be)